packages/lib-user/src/components/MyGroups/components/CreateButton/AddIcon.js

```diff
@@ -6,7 +6,7 @@ function AddIcon(props) {
       viewBox='0 0 15 15'
       {...props}
     >
-      <path d="M13.9286 9.07143H8.57143V14.4286C8.57143 15.0179 8.08929 15.5 7.5 15.5C6.91071 15.5 6.42857 15.0179 6.42857 14.4286V9.07143H1.07143C0.482143 9.07143 0 8.58929 0 8C0 7.41071 0.482143 6.92857 1.07143 6.92857H6.42857V1.57143C6.42857 0.982143 6.91071 0.5 7.5 0.5C8.08929 0.5 8.57143 0.982143 8.57143 1.57143V6.92857H13.9286C14.5179 6.92857 15 7.41071 15 8C15 8.58929 14.5179 9.07143 13.9286 9.07143Z" fill="#5C5C5C" />
+      <path d="M13.9286 9.07143H8.57143V14.4286C8.57143 15.0179 8.08929 15.5 7.5 15.5C6.91071 15.5 6.42857 15.0179 6.42857 14.4286V9.07143H1.07143C0.482143 9.07143 0 8.58929 0 8C0 7.41071 0.482143 6.92857 1.07143 6.92857H6.42857V1.57143C6.42857 0.982143 6.91071 0.5 7.5 0.5C8.08929 0.5 8.57143 0.982143 8.57143 1.57143V6.92857H13.9286C14.5179 6.92857 15 7.41071 15 8C15 8.58929 14.5179 9.07143 13.9286 9.07143Z" fill="currentColor" />
```

Comment:
Same question as above about attribution. Any reason this can't be `grommet-icon`'s Add icon?

Reply:
Great question regarding about attribution, I'm not sure.
The add icon is from the [related the Figma design](https://www.figma.com/file/qbqbmR3t5XV6eKcpuRj7mG/Group-Stats?node-id=829%3A6728&mode=dev). The Grommet Add icon is a little different.
Here's what it looks like as is:
![Add icon per Figma](https://github.com/zooniverse/front-end-monorepo/assets/12118751/9585877c-a84a-4de3-a0d8-ee0f0bda0919)

Here's what it would look like with grommet-icons:
![Add icon per grommet-icons](https://github.com/zooniverse/front-end-monorepo/assets/12118751/71bb0139-c105-48ca-987a-b5e4d4451e59)

Reply:
Grommet icon looks great.